tests/testthat/test_joins.R

```diff
@@ -0,0 +1,47 @@
+context("Test data joins")
+
+stripattr <- function(x, attrs = c('code', 'join_cols')) {
+  for (attr in attrs) attributes(x)[[attr]] <- NULL
+  x
+}
+
+d1 = readr::read_csv("~/iNZight/scripts/join.csv")
```

Comment:
Tests run in the `./tests/testthat` directory, so these only need to be
```suggestion
d1 <- readr::read_csv("join.csv")
```